apps/dashr-svm/app.R

```diff
@@ -0,0 +1,514 @@
+                          list(label = "Polynomial", value = "poly"),
+                          list(label = "Sigmoid", value = "sigmoid")
+                        ),
+                        value = "rbf",
+                        clearable = FALSE,
+                        searchable = FALSE
+                      ),
+                      namedSlider(
+                        name = "Cost (C)",
+                        id = "slider-svm-parameter-C-power",
+                        min = -2,
+                        max = 4,
+                        marks = as.list(setNames(10 ** (-2:4), -2:4)),
+                        value = 0
+                      ),
+                      formattedSlider(
+                        id = "slider-svm-parameter-C-coef",
+                        min = 1,
+                        max = 9,
+                        value = 1
+                      ),
+                      namedSlider(
+                        name = "Degree",
+                        id = "slider-svm-parameter-degree",
+                        min = 2,
+                        max = 10,
+                        value = 3,
+                        step = 1,
+                        marks = as.list(
+                          setNames(
+                            seq(2, 10, 2),
+                            seq(2, 10, 2)
+                          )
+                        )
+                      ),
+                      namedSlider(
+                        name = "Gamma",
+                        id = "slider-svm-parameter-gamma-power",
+                        min = -5,
+                        max = 0,
+                        value = -1,
+                        marks = as.list(setNames(10 ** (-5:0), -5:0))
+                      ),
+                      # formattedSlider is defined in utils/reusableComponents.R
+                      formattedSlider(
+                        id = "slider-svm-parameter-gamma-coef",
+                        min = 1,
+                        max = 9,
+                        value = 5
+                      ),
+                      # namedRadioItems is defined in utils/reusableComponents.R
+                      namedRadioItems(
+                        name = "Shrinking",
+                        id = "radio-svm-parameter-shrinking",
+                        labelStyle = list(
+                          marginRight = "7px",
+                          display = "inline-block"
+                        ),
+                        options = list(
+                          list(label = " Enabled", value = TRUE),
+                          list(label = " Disabled", value = FALSE)
+                        ),
+                        value = TRUE
+                      )
+                    )
+                  ),
+                  htmlDiv(
+                    dccMarkdown(
+                      paste0(
+                         "[Click here](https://github.com/plotly/dash-svm)",
```

Comment:
I think we might want to modify this to match the monorepo URL, since I believe the standalone repo will be going away soonish.

Reply:
Good point:
 
```
"[Click here](https://github.com/plotly/dash-sample-apps/tree/master/apps/dashr-svm)",
```

And I'll change the link at the top too:

```
htmlA(
  "Support Vector Machine (SVM) Explorer",
  href = "https://github.com/plotly/dash-sample-apps/tree/master/apps/dashr-svm"
...
```